### ISSUE TITLE
Still relevant?

### ISSUE THREAD

**Issue by jaredmoody** [justinweiss/bulk_cache_fetcher#4]
👋 

Given that there's not much activity on this gem I'm wondering if the approach is still relevant? Does it turn out in practice that it's most often worth it to just always eager load (or the reverse, ignore the n+1's) because there's not much to gain in most cases? Or is this gem just feature complete?

I was looking for anyone writing about the tradeoffs of eager loading (or not) with russian doll caching and this gem and your blog post about it are all I've found so far on this topic of dealing with n+1's with russian doll caching, so I'm curious.

Thanks!

**Comment by justinweiss** [comment]
Yep, the approach should still be relevant. The gem is extremely simple, so there shouldn't be much to add.

In the past few years, I haven't worked on websites that have a big public-facing component, which is where I find heavy russian doll caching most useful, so I haven't kept up with any changes that might be necessary --  but I also haven't heard of anything that still needs to be done!
